src/cli/uuid.test.ts

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { describe, expect, it, vi, beforeEach } from "vitest";
+import uuid, { getPeprDeploymentsByUUID, buildUUIDTable } from "./uuid";
+import { Command } from "commander";
+import type { kind } from "kubernetes-fluent-client";
+import { KubernetesListObject } from "@kubernetes/client-node";
+
+vi.mock("kubernetes-fluent-client", async () => {
+  const actual = await vi.importActual<typeof import("kubernetes-fluent-client")>(
+    "kubernetes-fluent-client",
+  );
+
+  const Get = vi.fn().mockResolvedValue({
+    items: [
+      {
+        metadata: {
+          name: "test-deployment",
+          labels: { "pepr.dev/uuid": "1234" },
+          annotations: { "pepr.dev/description": "Test annotation" },
+        },
+      },
+    ],
```

Comment:
I think these unit tests could benefit from a case where there are multiple deployments, so here's an example of an additional one to add.

```suggestion
      {
        metadata: {
          name: "other-deployment",
          labels: { "pepr.dev/uuid": "asdf" },
          annotations: { "pepr.dev/description": "Another annotation" },
        },
      }
    ],
```